ci_framework/roles/artifacts/tasks/main.yml

```diff
@@ -56,15 +56,14 @@
     - always
   ansible.builtin.import_tasks: cleanup.yml
 
-- name: Gather logs when forced or tests failed
+- name: Run must gather logs when forced or tests failed
   when: cifmw_artifacts_gather_logs | bool or not cifmw_success_flag.stat.exists
-  block:
-    - name: Run os_must_gather
-      ansible.builtin.import_role:
-        name: os_must_gather
-  rescue:
-    - name: Get CRC logs if os_must_gather failed
-      ansible.builtin.import_tasks: crc.yml
-  always:
-    - name: Get EDPM logs
-      ansible.builtin.import_tasks: edpm.yml
+  ansible.builtin.import_role:
+    name: os_must_gather
```

Comment:
The already exising block prevents the role to fail in case of error, now, if os_must_gather fails, the whole role and the logs collection playbook will fails. We should preserve the philosophy of the artifacts role to be safe to call, preventing raising errors.